go/engine/login_provision.go

```diff
@@ -566,6 +566,10 @@ func (e *LoginProvision) syncedPGPKey(ctx *Context) (libkb.GenericKey, error) {
 	return unlocked, nil
 }
 
+func (e *LoginProvision) noSyncedKeyMsg() string {
+	return fmt.Sprintf("ERROR!\n\nSorry, you have a PGP private key that Keybase doesn't have a copy of. You need to prove you're you. Any of these will unlock this computer:\n\n1. reset your account and start fresh: https://keybase.io/%s and click \"reset my keys\" in the gear\n2. install GPG and put your PGP private key on this machine and try again\n3. go back and provision with another device or paper key\n\n", e.user.GetName())
```

Comment:
Ok I think that makes sense.

On Thursday, January 7, 2016, Patrick Crosby <notifications@github.com>
wrote:

> In go/engine/login_provision.go
> <https://github.com/keybase/client/pull/1652#discussion_r49116528>:
>
> > @@ -566,6 +566,10 @@ func (e *LoginProvision) syncedPGPKey(ctx *Context) (libkb.GenericKey, error) {
> >  	return unlocked, nil
> >  }
> >
> > +func (e *LoginProvision) noSyncedKeyMsg() string {
> > +	return fmt.Sprintf("ERROR!\n\nSorry, you have a PGP private key that Keybase doesn't have a copy of. You need to prove you're you. Any of these will unlock this computer:\n\n1. reset your account and start fresh: https://keybase.io/%s and click \"reset my keys\" in the gear\n2. install GPG and put your PGP private key on this machine and try again\n3. go back and provision with another device or paper key\n\n", e.user.GetName())
>
> Will need a different error type then as libkb.NoKeyError is used for
> other missing keys as well and then the client output would be incorrect...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1652/files#r49116528>.
>